### PR TITLE
Added Logo and Updated proposals

### PR DESCRIPTION
**Pull request by romitkarmakar**
Resolved Issues #29, #33 , #24

### CODE REVIEW

**Review by AvinashAgarwal14 — DISMISSED** [review]

**Comment by ZER-0-NE** [comment]
@nkprince007 @pandusonu2 @djmgit , please review.

**Comment by pandusonu2** [comment]
Can you also add the google doc link on the page? Would be helpful for other GSoCers to update later on, if they haven't, just a small link to the top right corner would be fine

**Comment by romitkarmakar** [comment]
Added this changes

> Can you also add the google doc link on the page? Would be helpful for other GSoCers to update later on, if they haven't, just a small link to the top right corner would be fine

File: GSoC_resources.md [hunk 1]
```diff
@@ -6,67 +6,65 @@ permalink: gsoc_resources
...
+* Sudhanshu Saurav
+* [Lokesh Nandanwar/So-Sy Labs](https://summerofcode.withgoogle.com/archive/2018/projects/4726926518255616/)
+* [Akshat Garg/FOSSASIA](https://summerofcode.withgoogle.com/archive/2018/projects/4537155267330048/)
+* [Hrishav Mukherjee/Boston University](https://summerofcode.withgoogle.com/archive/2018/projects/5687777542799360/)
 
-* [FOSSASIA](http://labs.fossasia.org/ideas.html)
+#### GSoc 2017
 
-* [OpenSUSE](http://101.opensuse.org/)
+* Naveen
```
Comment (naveenkumarsangi) [inline GSoC_resources.md]: `[Naveen Kumar Sangi](https://summerofcode.withgoogle.com/archive/2017/projects/4690491102724096/)`

Comment (naveenkumarsangi) [reply GSoC_resources.md]: Don't ignore changes and ask for a new review, please. @romitkarmakar

File: GSoC_resources.md [hunk 2]
```diff
@@ -6,67 +6,65 @@ permalink: gsoc_resources
...
+* Compro
+* Deepjyoti
 
-* [SciRuby](https://github.com/SciRuby/sciruby/wiki/Google-Summer-of-Code-2017-Ideas)
+#### GSoC 2016
 
-## Previous years selected proposals
+* [sonalkr132/rubygems.org](https://gist.github.com/sonalkr132/148875c61360e3349687ff05df04db2d)
+* [ananyo2012/publiclab](https://publiclab.org/notes/ananyo2012/03-25-2016/expanded-q-a-system-for-publiclab-org)
+* Vamsi
```
Comment (pandusonu2) [inline GSoC_resources.md]: `[Vamsi](https://summerofcode.withgoogle.com/archive/2016/projects/5194537727164416/)`

I thought I filled the google form, anyways, here is mine 😬

File: GSoC_resources.md [hunk 3]
```diff
@@ -6,67 +6,65 @@ permalink: gsoc_resources
 
 All of us go through the same process of devouring the internet when we are writing our proposal for GSoC. This is an attempt to bookmark all those resources in one place.
 
-## GSoC 2017 Ideas page
+Please fill this [GSoC accepted proposal form](https://forms.gle/xaoXmpEdeAJ9c9Fn9), so that it could help other potential GSoC candidates to write a more competent proposal.
 
+## This year selected proposals
+### GSoC 2019
```
Comment (pandusonu2) [inline GSoC_resources.md]: I would prefer the heading to be "Selected projects", no need to mention "this" and "previous" sections, as "this year" would be relevant only for about 4 months of the yr at best.

**Comment by pandusonu2** [comment]
Thanks for the changes @romitkarmakar :)

Can I also know, who has the access to the submissions on the google doc?

**Comment by ZER-0-NE** [comment]
@pandusonu2 that would be me. I'll add you as a collaborator if you'd like.

**Comment by pandusonu2** [comment]
@ZER-0-NE export it to an excel and have it be a link sharing non-editable document?

**Comment by ZER-0-NE** [comment]
This would do it: https://docs.google.com/spreadsheets/d/1n3bqanFoBcSsrBtjvpVW_NMgyf3sOUDxNAchwM-t4zs/edit?usp=sharing
@AvinashAgarwal14 @romitkarmakar

**Comment by pandusonu2** [comment]
@romitkarmakar include that in the page as well? (/.\) Will be easier for later

File: GSoC_resources.md (hunk 3 above)
Comment (romitkarmakar) [reply GSoC_resources.md]: Removed this year selected proposals

File: GSoC_resources.md (hunk 2 above)
Comment (romitkarmakar) [reply GSoC_resources.md]: Added this link.

File: GSoC_resources.md (hunk 1 above)
Comment (romitkarmakar) [reply GSoC_resources.md]: Added this link.

File: GSoC_resources.md [hunk 4]
```diff
@@ -6,67 +6,66 @@ permalink: gsoc_resources
...
-* [BeagleBoard](http://elinux.org/BeagleBoard/GSoC/Ideas)
-
-* [Eclipse](https://wiki.eclipse.org/Google_Summer_of_Code_2017_Ideas)
-
-* [OSGeo](https://wiki.osgeo.org/wiki/Google_Summer_of_Code_2017_Ideas)
-
-* [redhenlab](http://www.redhenlab.org/summer-of-code/summer-of-code-2017---ideas-page)
-
-* [Ruby](https://github.com/rubygsoc/rubygsoc/wiki/Ideas-List)
+## Previous years selected proposals
```
Comment (pandusonu2) [inline GSoC_resources.md]: As I said, remove this line

File: GSoC_resources.md [hunk 5]
```diff
@@ -6,67 +6,66 @@ permalink: gsoc_resources
...
+* [Abhishek Singh/ESIP](https://summerofcode.withgoogle.com/projects/#4567128563253248)
+* [Mahesh Tudu @ FreeBSD](https://summerofcode.withgoogle.com/projects/#6555942194249728)
+* [Romit Kumar @ BRL-CAD](https://summerofcode.withgoogle.com/projects/#5042363104231424)
+* [Jayjeet Chakraborty @ CROSS](https://summerofcode.withgoogle.com/projects/#6075586542305280)
+* [Avinash Aggarwal/SugarLabs](https://summerofcode.withgoogle.com/projects/#5993905961566208)
+* [Ayan Banerjee @ UCSC Xena](https://summerofcode.withgoogle.com/projects/#5795682349219840)
+* [Sudipta Pandit @ The Honeynet Project](https://summerofcode.withgoogle.com/projects/#6189934577188864)
 
-* [libvirt - virtualization API](https://wiki.libvirt.org/page/Google_Summer_of_Code_Ideas)
+[Check GSoC Google Sheets](https://docs.google.com/spreadsheets/d/1n3bqanFoBcSsrBtjvpVW_NMgyf3sOUDxNAchwM-t4zs/edit?usp=sharing)
```
Comment (pandusonu2) [inline GSoC_resources.md]: Should be above along with the form link

File: Gemfile.lock [hunk 6]
```diff
@@ -5,6 +5,7 @@ GEM
       public_suffix (>= 2.0.2, < 4.0)
     colorator (1.1.0)
     ffi (1.10.0)
+    ffi (1.10.0-x64-mingw32)
```
Comment (pandusonu2) [inline Gemfile.lock:8]: @nkprince007 to check this part 😬 
I have no idea

Comment (naveenkumarsangi) [reply Gemfile.lock:8]: Looks to me like some windows specific dependency. Let it stay for now.

**Review by naveenkumarsangi — COMMENTED** [review]
I'm okay with that. But look into fixing the changes suggested by @pandusonu2

File: GSoC_resources.md (hunk 4 above)
Comment (romitkarmakar) [reply GSoC_resources.md]: Removed this line

**Review by pandusonu2 — APPROVED** [review]

**Comment by pandusonu2** [comment]
Merged it :) 

Thanks for your contribution @romitkarmakar

**Pull request merged**